### PR TITLE
Move System.Drawing.Common to ToolsetDependencies

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -368,6 +364,10 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>bcb608e3d21344a0bb7c3399f060143504bbf475</Sha>
     </Dependency>
+    <Dependency Name="System.Drawing.Common" Version="8.0.0-preview.7.23356.8">
+      <Uri>https://github.com/dotnet/runtime</Uri>
```
Comment (mmitche) [inline eng/Version.Details.xml:368]:
```suggestion
      <Uri>https://github.com/dotnet/winbforms</Uri>
```

File: eng/Version.Details.xml [hunk 2]
```diff
@@ -368,6 +364,10 @@
...
+      <Sha>5d54b08d5fc40d0b1c156f430a487a94c1e34f79</Sha>
```
Comment (mmitche) [inline eng/Version.Details.xml:369]:
```suggestion
      <Sha>906ac1628112861933c3c00874cdef4621d13b36</Sha>
```

Comment (wtgodbe) [reply eng/Version.Details.xml:369]: Is the version number right?

File: eng/Version.Details.xml (hunk 1 above)
Comment (wtgodbe) [inline eng/Version.Details.xml:368]:
```suggestion
      <Uri>https://github.com/dotnet/winforms</Uri>
```

**Review by mmitche — APPROVED** [review]

**Review by mmitche — APPROVED** [review]

**Pull request merged**